libs/components/core/src/lib/modules/affix/affixer.spec.ts

```diff
@@ -14,13 +15,65 @@ import { SkyAffixer } from './affixer';
 
 @Component({
   template: `
-    <div class="affixer-container">
-      <div #baseElement class="affixer-base">base</div>
+    <div
+      class="affixer-layer"
+      [ngClass]="{
+        'affix-layer-fixed': fixedLayer1,
+        'affix-layer-overflow': overflowLayer1
+      }"
+    >
+      <div
+        class="affixer-layer"
+        [ngClass]="{
+          'affix-layer-fixed': fixedLayer2,
+          'affix-layer-overflow': overflowLayer2
+        }"
+      >
+        <div
+          class="affixer-layer"
+          [ngClass]="{
+            'affix-layer-fixed': fixedLayer3,
+            'affix-layer-overflow': overflowLayer3
+          }"
+        >
+          <div class="affixer-container">
+            <div #baseElement class="affixer-base">base</div>
+          </div>
+          <div #affixedElement class="affixed">affixed</div>
+        </div>
+      </div>
```

Comment:
The new issue was an affixed country field within an `overflow: hidden` element. These test assert the behavior.